index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html>
```

Comment:
HTML structure should be: 
```suggestion
<!DOCTYPE html>
<html>
<head>
*put title, metatags, links to css* 
</head>
<body>
*semantic HTML, any visual for user to see such as txt, images, videos etc* 
</body>
</html>
```